### ISSUE TITLE
vendor/autoload.php is missing from sources?

### ISSUE THREAD

**Issue by nimrod-cohen** [iboldurev/api-ai-php#7]
I'm trying to use this package without composer's assistance.

Should I "require" each and every file separately?
Also, when doing that, the code complains that GuzzleHttp is missing - is this a separate package I should download and require?

thanks in advance,
Nimrod.

**Comment by Smus** [comment]
Did you find the file? im missing it aswell :-(

**Comment by nimrod-cohen** [comment]
No, I ended up communicating with API.AI directly, which turned out to be an easy thing.

I'm using wordpress which has a really good http calls class

**Comment by nimrod-cohen** [comment]
if this is any help:
	$headers = ["Authorization" => "Bearer ".get_option("aisupport_token"),
					"Content-Type" => "application/json; charset=utf-8"];
		$data = [
			"query" => $question,
			"v" => "20170519", //current date of development
			"lang" => "en",
			"sessionId" => $_SESSION["aisupport_session"]
		];
		$result = wp_remote_post("https://api.api.ai/v1/query",
			[
				'headers'   => $headers,
				'body'      => json_encode($data)
			]);
		return $result["body"];

**Comment by gambry** [comment]
vendor/autoload.php is generated by composer, no composer means no autoload.
You need to come up with your own solution for loading this library if you don't use composer.

**Comment by ilyasahsan123** [comment]
you should install on composer sir
